Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -321,17 +323,26 @@ public override Task OpenAsync(CancellationToken cancellationToken)
                 cancellationToken);
         }
 
-        public Mutex GetArrayBindingMutex()
+        public string GetQueryId()

```

Comment:
why do we need on connection?

Reply:
That's just in case that application might have the DbCommand instance for previous query execution destroyed but need to get the last query id.

Reply:
If other driver don't have at connection, then please remove it. Otherwise it will seen as feature GAP.

Reply:
I see, will remove.

Reply:
done